arviz/plots/densityplot.py

```diff
@@ -17,6 +17,7 @@ def plot_density(
     group="posterior",
     data_labels=None,
     var_names=None,
+    combine_dims=[],
```

Comment:
The default value should be `None`. Using mutable objects like lists as default arguments can have unexpected effects, see for example https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument

Reply:
I believe this is resolved -- I changed the default to be `None`, but I believe `skip_dims` still requires a set-like object, so I added in a line that passes in an empty list instead, if `skip_dims = None`.